components/DraggableItem.vue

```diff
@@ -77,9 +77,8 @@ export default {
 
       if (event.relatedTarget) {
         // dropped on dropzone!
-        const dropzoneWidth = interact.getElementRect(event.relatedTarget).width
         const cardWidth = boundingClientRect.width
-        const value = this.screenX / (dropzoneWidth - cardWidth)
+        const value = this.screenX / (window.innerWidth - cardWidth)
```

Comment:
DropzoneBackground component is intentionally wider than the screen(width: ~110%) and so is the dropzone. This was messing up the accuracy, thus the change. 

The DropzoneBackground has to be wider than screen so that it fills the screen from one end to the other with the blur filter style applied.

Reply:
ok, that may be, but this is not an elegant way to solve it.
We have two different components: 
`DropZone`: responsible for Drag&Drop logic, including the calculation of `value`
and within it 
`DropZoneBackground`: responsible for aesthetics, including the proper display of the gradient.

I strongly prefer to keep the `DropZone` logic and `value` calculation correct and keep aesthetic issues and solutions contained in `DropZoneBackground`. 
What if `DropZoneBackground` would use tricks like negative horizontal margin to make the gradient display properly?

Reply:
I agree with you, but i can't think of any quick css fix.

The problem is not about positioning of the gradient, margins won't help. The dropzone background has to be wider than the screen because the blur effect shortens the gradient effect. When it's longer the 'shortening' is hidden beyond the bounds of the screen.

A more elegant way to implement this without breaking the logic you're describing would be to implement the gradient effect in a canvas.

Reply:
![Screenshot 2022-05-18 at 08 46 34](https://user-images.githubusercontent.com/18234033/168975217-a5d8c33e-6c74-457d-9c82-6b8b96c015bf.png)

This is the result of 100% wide blurred gradient

Reply:
is this shortening a bad thing?